src/components/pages/program-view/ProgramViewContainer.jsx

```diff
@@ -17,7 +18,7 @@ const Container = styled.div`
 const programData = {
```

Comment:
I would personally recommend adding a `dummydata.js` file into your program-view folder just to keep this `ProgramViewContainer` component really tight and clean. That way once the backend is ready to roll, we can barely change anything in this component and just delete the file!!

Reply:
this is a solid mockout of what the API data will look like, though — really fantastic job on that.

Reply:
Awesome comments Chaz, thanks so much for taking the time to look though my code and give such thoughtful, clear and actionable feedback! I totally agree with all of your assesments here re: Math.random, using courseId and moduleId instead as well as pulling the test data into a side dummydata.js file. Will make those changes shortly!